python/mpi_cmake_modules/cmake_format.py

```diff
@@ -0,0 +1,78 @@
+""" @namespace mpi_cmake_modules.cmake_format.py
+
+Formatting script based on cmake-format
+
+License BSD-3-Clause
+Copyright (c) 2019, New York University and Max Planck Gesellschaft.
```

Comment:
It's 2021 now :) (same on other new files)

Reply:
Isn't it the year of creation and then implied "and above"?

Reply:
Ah, you're considering the package as a whole?  Then this makes sense.  I was thinking on a per-file base.  I'm not sure if the year here actually has any legal relevance, though.